webapp/src/index.tsx

```diff
@@ -137,7 +152,8 @@ export default class Plugin {
         }
 
         registry.registerAdminConsoleCustomSetting('Config', Config);
-        if (rhs) {
+
+        if (rhs && !registry.registerAppBarComponent) {
```

Comment:
added this for backward compatibility, not sure if actually needed.

Reply:
We would need to actually have the backwards compatibility code up there, but yes it would be nice for now to have it.